pkg/serverless/trace/inferredspan/span_enrichment_test.go

```diff
@@ -201,6 +202,57 @@ func TestEnrichInferredSpanWithSNSEvent(t *testing.T) {
 	assert.True(t, inferredSpan.IsAsync)
 }
 
+func TestEnrichInferredSpanWithSQSEvent(t *testing.T) {
```

Comment:
Is this supposed to be here? Looks like this is a PR for DynamoDB.

Reply:
I saw we had missed it previously so I just added it here

Reply:
Sounds good!